src/fparser/two/Fortran2003.py

```diff
@@ -4323,9 +4323,9 @@ class Primary(Base):  # R701
                 | ( <expr> )
     """
     subclass_names = [
-        'Constant', 'Parenthesis', 'Designator', 'Array_Constructor',
+        'Constant', 'Designator', 'Array_Constructor', 
```

Comment:
Looks like there is a spurious space at the end of this line. This should cause pycodestyle and/or pylint to complain.